resources/themes/Night Mapping/style.qss

```diff
@@ -226,6 +226,7 @@ QLineEdit
     padding: 0.12em;
     border: 1px solid @itemdarkbackground;
     background-color: @itembackground;
+    color: @text;
```

Comment:
This can't be added back, it was removed to insure placeholder styling worked for line edits.

Reply:
Unfortunately, this fixes Advanced Digitizing Panel on Windows as it seems:

![grafik](https://github.com/qgis/QGIS/assets/78353871/bb9b91dd-9f32-4117-9f41-b10efe99b502)

Reply:
@nirvn: You are certainly much more experienced with QML than I am, but I can't find any problems with the placeholders on either latest Windows or Linux. Maybe its now fixed in QT? https://bugreports.qt.io/browse/QTBUG-92199 

However, as far as I know there are also QGIS installations with < QT 5.15.12... Would you consider it possible to implement the workaround described there in QGIS with reasonable effort?